### PR TITLE
Fix typo modifier onlyowner using all lowercase instead of mixed case

### PR DESCRIPTION
**Pull request by ebouchut**
Update the style guide to fix the modifier name `onlyowner` using lowercase where mixedCase is expected.

### CODE REVIEW

File: docs/style-guide.rst [hunk 1]
```diff
@@ -560,7 +560,7 @@ Yes::
         return x + 1;
     }
 
-    function increment(uint x) public pure onlyowner returns (uint) {
+    function increment(uint x) public pure onlyOwner returns (uint) {
```
Comment (cameel) [inline docs/style-guide.rst:563]: Thanks for the correction! I see that there are a few more instances of lowercase `onlyowner` in this file (and they're not there to illustrate the style that's not recommended). Could you fix them too?

Comment (ebouchut) [reply docs/style-guide.rst:563]: Hi @cameel, ✅ Indeed, I added commit a90bf95fcb081f999019b4a30b96968962710a48 with the remaining typos fixed.

**Review by cameel — DISMISSED** [review]
Looks fine. Could you also rebase it on top of develop?

**Review by chriseth — APPROVED** [review]

**Comment by ebouchut** [comment]
Hi  @cameel and @chriseth,
I apologize for being late to the party :-).
Thanks for the force push in the meantime.

**Pull request merged**